networkx/utils/tests/test_rcm.py

```diff
@@ -34,7 +34,7 @@ def test_rcm_alternate_heuristic():
     answers = [[6, 3, 5, 7, 1, 2, 4, 0], [6, 3, 7, 5, 1, 2, 4, 0]]
 
     def smallest_degree(G):
-        node, deg = min(G.degree().items(), key=lambda x: x[1])
+        deg, node = min((d, n) for n, d in G.degree())
```

Comment:
Simpler: `return min(G, key=G.degree)`. This matches the example given in the documentation in `utils.rcm`.